packages/apollo-datasource-rest/src/RESTDataSource.ts

```diff
@@ -95,6 +95,7 @@ export abstract class RESTDataSource<TContext = any> extends DataSource {
   protected async didReceiveResponse<TResult = any>(
     response: Response,
     _request: Request,
+    _context: TContext,
```

Comment:
It's great that `TContext` already exists here!

I think this should be an optional parameter to account for the existing pattern?
```suggestion
    _context?: TContext,
```